src/Item/PluginItem/Interpretations/Interpretation.js

```diff
@@ -127,6 +139,42 @@ class Interpretation extends Component {
         this.setState({ showCommentField: false });
     };
 
+    editComment = (commentId, text) => {
+        const { id } = this.props.interpretation;
+        id === commentId
+            ? this.props.updateInterpretation({ id, text })
+            : this.props.updateInterpretationComment({
+                  id,
+                  commentId,
+                  text,
+              });
+        this.toggleEdit(commentId);
+    };
+
+    renderCommentOrEditField = item => {
+        return this.state.editing.includes(item.id) ? (
+            <InputField
+                editing
+                placeholder={i18n.t(item.text)}
+                commentId={item.id}
+                postText={i18n.t('Post')}
+                onEdit={this.editComment}
+            />
+        ) : (
+            <p style={style.text}>{i18n.t(item.text)}</p>
```

Comment:
@joakimia item.text is the actual comment, so there is no translation for it. So remove the i18n.t part.